ui/apps/dashboard/src/app/(organization-active)/(dashboard)/settings/integrations/integrations.tsx

```diff
@@ -31,21 +31,11 @@ const INTEGRATIONS: Integration[] = [
     title: 'Vercel',
     slug: 'vercel',
     Icon: <IconVercel className="text-onContrast h-6 w-6" />,
-    actionButton: ({ enabled, hasError, isMarketplace }) => {
+    actionButton: ({ enabled, hasError }) => {
       let label: string;
       let target = undefined;
       let url: string;
-      if (isMarketplace) {
-        // Marketplace integration installed.
-
-        label = 'Manage';
-        target = '_blank';
-
-        // Managed in Vercel's dashboard.
-        url = 'https://vercel.com/integrations/inngest';
```

Comment:
does this mean we can't manage it from vercel's dashboard? or we can do it from either?